### PR TITLE
Updating TestRunner to sort unit test assemblies by size.

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @jaredpar, @ManishJayaswal, @jmarolf, @heejaechang 

This is a partial fix. I will be merging the internal and external test runners (where appropriate) and getting these sorted based off time (hopefully later today).

### CODE REVIEW

File: netci.groovy [hunk 1]
```diff
@@ -71,7 +71,7 @@ static void addUnitPublisher(def myJob) {
       'xunit'('plugin': 'xunit@1.97') {
       'types' {
         'XUnitDotNetTestType' {
-          'pattern'('**/*TestResults.xml')
+          'pattern'('**/xUnitResults/*.xml')
```
Comment (jaredpar) [inline netci.groovy]: Why this change?

Comment (jaredpar) [inline netci.groovy]: Won't this be different between our public and internal tests?  

Alos I'd separate this out. We already need to regenerate jobs today for other changes we're making.  Lets combine it all together and do it at once.

Comment (tannergooding) [inline netci.groovy]: Because our internal tests already separate logs out into a separate folder. This also makes them easier to locate on your box :)

Also the job regeneration is cheap (it takes less than a minute and updates all the jobs simultaneously). It shouldn't matter if we update it multiple times.
